### DISCUSSION TITLE
Remove a category

### DISCUSSION THREAD

**Discussion by lamyergeier** [FreshRSS/FreshRSS#7102] [Q&A] [answered]
How to remove a category and is it also possible to rename a catgory?

**Comment by math-GH** [comment, marked as answer]
Renaming:
Open the config of a category:
![grafik](https://github.com/user-attachments/assets/a492f0e1-41c7-4dd0-b05b-4fbf7cf90a3c)


Remove:
Open the config of a category and scroll a bit down to the delete buttons:
![grafik](https://github.com/user-attachments/assets/ced44f85-641c-4b1d-ad5f-dd1e53fd454f)
